### ISSUE TITLE
Touchables sporadically invoke onPress within scrolling ScrollView

### ISSUE THREAD

**Issue by dcalhoun** [necolas/react-native-web#924]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Touchables within `ScrollView` invoke `onPress` handler when I expect the `ScrollView` to prevent it. In my test case when the `onPress` handler is invoked, the Tile (Touchable) turn orange and the "Tiles Pressed" count is also incremented.

**If the current behavior is a bug, please provide the steps to reproduce and
if a minimal demo of the problem via Glitch or similar (template:
https://glitch.com/edit/#!/react-native-web-playground).**

1. Visit the [test case](https://rnw-touchable-test-wynueewgdx.now.sh) ([source](https://github.com/HealthTeacher/rnw-touchable-test/blob/master/index.js)).
2. Scroll through the list of Tiles flicking at different rates ([video](https://streamable.com/enksr)).

-or-

1. Visit the [test case](https://rnw-touchable-test-wynueewgdx.now.sh) ([source](https://github.com/HealthTeacher/rnw-touchable-test/blob/master/index.js)).
2. At the top of the list, touch a Tile at the top and scroll upwards and release ([video](https://streamable.com/53m18)).

**What is the expected behavior?**
Regardless of speed of scroll, the Tile (Touchable) should not register a `onPress` event when scrolling the `ScrollView`.

**Environment (include versions). Did this work in previous versions?**

* OS: iOS
* Device: iPhone 7
* Browser: Safari 11.1
* React Native for Web (version): 0.6.1
* React (version): 16.3.2

---

### Thoughts

- Maybe I implemented `ScrollView` incorrectly?
- Maybe I'm just bad at scrolling? I don't feel like what I'm doing is all that abnormal, but it's hard to showcase what a "normal" touch/swipe/scroll is remotely.
- My long-term goal is more [complex test case with nested `ScrollView`s](https://rnw-touchable-test-wynueewgdx.now.sh/nested.html) ([source](https://github.com/HealthTeacher/rnw-touchable-test/blob/master/nested.js)).

**Comment by dcalhoun** [comment]
Edited the post to add a couple of example videos for each test case.

**Comment by dcalhoun** [comment]
I was able to resolve this in my app, but I haven't determined what exactly I changed. Once I do I'll post an update for posterity. Sorry for the noise.

**Comment by rawris** [comment]
HI! @dcalhoun Can you please tell me how you resolved your issue?

**Comment by dcalhoun** [comment]
@rawris I do not have a specific fix to share with you, but one piece of advice I'd share with you is to make sure the `body` itself is not scrolling.

In some of my debugging, I noticed the `body` was scrolling and not the `ScrollView` I implemented. It is quite easy to notice this in iOS, as Safari's address bar will collapse as you scroll down the page. If you see this happening, you are scrolling the `body` and not the `ScrollView`. If that is occurring, `Touchable` will not function as you expect.
